google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ExternalTableDefinition.java

```diff
@@ -351,6 +381,9 @@ public HivePartitioningOptions getHivePartitioningOptions() {
   com.google.api.services.bigquery.model.Table toPb() {
     Table tablePb = super.toPb();
     tablePb.setExternalDataConfiguration(toExternalDataConfigurationPb());
+    if (getMaxStaleness() != null) {
```

Comment:
Does getMaxStaleness() construct a String or just return a reference? If it is constructing, it might be a good idea to store the result in a variable instead of calling the function twice in a row in the case that getMaxStaleness() does not return null. I don't think it matters much if the function just returns a reference.